packages/client/src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,35 +1,105 @@
+import * as React from 'react';
+import { Card, CardMedia, Typography, Rating, Box } from '@mui/material';
+import styled from '@emotion/styled';
 import FavoriteIcon from '@mui/icons-material/Favorite';
 import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
-import { Box, Card, CardMedia, Rating, Typography } from '@mui/material';
-import * as React from 'react';
 import { useFavorites } from './useFavorites'; // created a custom hook to manage favorites
 
 const BasicRating = () => {
-  return (
-    <Rating
-      name="read-only"
-      value={4}
-      readOnly
-      sx={{
-        position: 'relative',
-        left: '1.96rem',
-        borderRadius: '0.59',
-        maxHeight: 'min-content',
-        width: '3rem',
-        top: '2rem',
-        paddingBottom: '4rem',
-        color: '#003E2F',
-      }}
-    />
-  );
+  const StyledRating = styled(Rating)({
+    position: 'relative',
+    left: '1.96rem',
+    borderRadius: '0.59rem',
+    maxHeight: 'min-content',
+    width: '3rem',
+    top: '2rem',
+    paddingBottom: '4rem',
+    color: '#003E2F',
+  });
+  return <StyledRating name="read-only" value={4} readOnly />;
 };
 
-export const MovieCard = ({ movie }) => {
+export function MovieCard({ movie }) {
   const [favorites, toggleFavorite] = useFavorites([]);
   const isFavorite = favorites.includes(movie);
 
+  const StyledCard = styled(Card)`
+    display: flex;
+    flex-direction: column;
+    max-height: min-content;
+    width: 17.43rem;
+    left: 6rem;
+    background: linear-gradient(
+      166.18deg,
+      rgba(0, 0, 0, 0.4) 0%,
+      rgba(0, 0, 0, 0) 98.81%
+    );
+    box-shadow: 20px 40px 70px rgba(0, 0, 0, 0.25);
+  `;
+
+  const sharedIconStyles = `
+  display: block;
+  position: absolute;
+  width: 1.9rem;
+  height: 1.75rem;
+  top: 2.25rem;
+  right: 1.5rem;
+  color: #ff0000;
+  padding: 1rem;
+  background-color: #000000cc;
+  z-index: 1;
+  &:hover: {
+    color: #ff0000;
+    cursor: pointer;
+  }
+`;
+
+  const StyledFavoriteIcon = styled(FavoriteIcon)`
+    ${sharedIconStyles}
+  `;
+
+  const StyledFavoriteBorderIcon = styled(FavoriteBorderIcon)`
+    ${sharedIconStyles}
+  `;
+  const StyledCardMedia = styled(CardMedia)`
+    position: relative;
+    max-height: min-content;
+    width: 13.5rem;
+    left: 0;
+    top: 0;
+    padding-top: 2.25rem;
+    padding-left: 1.96rem;
+  `;
+
+  const StyledTypographyTitle = styled(Typography)`
+    position: relative;
+    max-height: min-content;
+    width: 13.45rem;
+    top: 1.5rem;
+    left: 1.96rem;
+    font-family: Inter;
```

Comment:
To be safe, I would suggest to define a fallback font in case for some reason Inter is not loaded, such as Arial, serif, sans-serif etc. 

The way to do it is basically by listing the other fonts, comma separated. See https://www.w3schools.com/cssref/pr_font_font-family.php